documentation/modules/ROOT/pages/transformations/index.adoc

```diff
@@ -0,0 +1,18 @@
+= Transformations
+
+Connectors can be configured with transformations to make lightweight message-at-a-time modifications. {prodname} provides several single message transformations (SMTs) that you can use to modify event records before Kafka Connect saves the records to Kafka topics.
+
+We currently have the following {prodname} provided SMTs to use with Kafka Connect:
+
+* xref:transformations/topic-routing.adoc[Topic Routing]
```

Comment:
@gunnarmorling do you want this to be a table with sort description?

Reply:
wdym by "sort description"? Sort *direction* by any chance? If so, no I don't think it's needed.

Reply:
sry my bad its typo, I meant to arrange these available SMT's in a tabular form with a column of "short" description.

Reply:
Ah yes, a short description is a good idea 👍